eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/EventPosition.java

```diff
@@ -46,12 +46,12 @@ private EventPosition(boolean isInclusive) {
 
     /**
      * Returns the position for the start of a stream. Provide this position in receiver creation
-     * to start receiving from the first available event in the partition.
+     * to start receiving from the first available (earliest) event in the partition.
      *
      * @return An {@link EventPosition} set to the start of an Event Hubs stream.
      */
-    public static EventPosition firstAvailableEvent() {
-        return FIRST_AVAILABLE_EVENT;
+    public static EventPosition earliest() {
```

Comment:
Was this change motivated for consistency reasons across other languages?

Reply:
This change was motivated from feedback from the Service team this morning. Similar changes will be done for other languages as well